waku/v2/node/config.nim

```diff
@@ -155,23 +155,24 @@ type
   
     rlnRelayEthAccount* {.
       desc: "Account address for the Ethereum testnet Goerli", 
+      # NOTE: This can be derived from the private key, but kept for future use
       defaultValue: ""
-      name: "eth-account-address" }: string
+      name: "rln-relay-eth-account-address" }: string
```

Comment:
```suggestion
      name: "rln-relay-eth-account" }: string
```

Reply:
Addressed in [0395e29](https://github.com/status-im/nwaku/pull/1204/commits/0395e29b0706bca9d58705bea5e6e41e6f27afaf)